src/main/java/org/schemaspy/output/diagram/SummaryDiagram.java

```diff
@@ -0,0 +1,27 @@
+package org.schemaspy.output.diagram;
+
+import java.io.File;
+
+public class SummaryDiagram {
```

Comment:
Seems to be missing some abstraction here since this and TableDiagram are identical

Reply:
Might be good to look at #918 and also DiagramElement, would be nice if we could remove DiagramResult.

Reply:
Correct (discounting the `DiagramException` message). Together with `OrphanDigram`, they existed in parallel in `DiagramFactory`. The intention of this refactoring was to eliminate that class - so I propose that introducing an abstraction would be its own PR.